### PR TITLE
Add example setups for argo, etcd, hadoop, hbase, jetty, mongo, scyll…

### PR DESCRIPTION
**Pull request by yqlu**
…a, solr, tomcat, varnish

### CODE REVIEW

**Review by mtabasko — APPROVED** [review]
LGTM

File: examples/mongodb/exporter.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: mongodb-exporter
+    spec:
+      containers:
+      - name: exporter
+        image: percona/mongodb_exporter:0.20
+        args:
+        - --mongodb.uri=mongodb://user:password@mongodb-service-name:27017
```
Comment (lyanco) [inline examples/mongodb/exporter.yaml:35]: Doesn't this line need a +? user:password should need to be changed

**Review by lyanco — CHANGES_REQUESTED** [review]

File: examples/mongodb/exporter.yaml (hunk 1 above)
Comment (mtabasko) [reply examples/mongodb/exporter.yaml:35]: Probably -- which means we'll have to change this to a "snippet" file here and in the import. (No biggie, just noting it).

Comment (yqlu) [reply examples/mongodb/exporter.yaml:35]: The rule is slightly different: we've been using `+` to mark when you need to *add* lines to your *existing* application yaml to add a sidecar or turn on exporting by setting env variables.

There are two prior cases where we've instructed you to install a brand new standalone exporter -- we haven't been using `+`, but instead spell out in a paragraph following the yaml how the exporter should talk to the application.

[Elasticsearch](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/elasticsearch)

> The Elasticsearch exporter is configurable with [flags](https://github.com/prometheus-community/elasticsearch_exporter#configuration), which can be set as container args. You can adjust your exporter's configuration to suit your Elasticsearch configuration. Update the `--es.uri` flag to match your environment's username, password, and clusterIP service name.

[Kafka](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/kafka)

> The Kafka exporter is configurable with [flags](https://github.com/danielqsj/kafka_exporter#flags), which can be set as container args. This example assumes Kafka is available as a ClusterIP service named `kafka` on port `9092`.

**Review by lyanco — APPROVED** [review]

**Pull request merged**